redis/src/cluster_async/mod.rs

```diff
@@ -1355,6 +1380,24 @@ where
                         },
                     }));
                 }
+                Next::RetryBusyLoadingError {
+                    request,
+                    identifier,
+                } => {
+                    // TODO - do we also want to try and reconnect to replica if it is loading?
```

Comment:
I think that waiting for the next refresh slots is good enough since the loading can take minutes